### DISCUSSION TITLE
Has anyone encountered this?: Job data for "send-email" is too long

### DISCUSSION THREAD

**Discussion by Linny3D** [vendurehq/vendure#1513] [Q&A] [answered]
Yesterday, while testing the EMail Plugin in Vendure, i encountered this:

`Job data for "send-email" is too long to store with the mariadb driver (92kb).`

`The following keys were truncated:  
[run:*server] error 9.4.22, 11:24 - [Vendure Server] Could not add Job to "send-email" queue
[run:*server] QueryFailedError: ER_DATA_TOO_LONG: Data too long for column 'data' at row 1`

This seems to happen, if a customer has (in our case) more than 5 articles in the Cart. With fewer articles this does not happen.

Has anyone seen this error before? Don't know if its a bug justifying an issue, or if we are doing something wron.

**Comment by michaelbromley** [comment, marked as answer]
Hi,
This is a known issue. You can read a bit more about it [in this comment](https://github.com/vendure-ecommerce/vendure/blob/5017622ba224d43064d36fd43d8193fea1652482/packages/core/src/plugin/default-job-queue-plugin/sql-job-queue-strategy.ts#L48). 

The options you have are to use a different DB (postgres) or a different JobQueueStrategy, e.g. using the BullMQJobQueueStrategy which stores the job data in Redis rather than in the DB (it is also much more performant).

**Comment by Linny3D** [comment]
Thank you very much. That did the trick :)
